### PR TITLE
Move damemi to sig-scheduling reviewers

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Move @damemi back to reviewer. Mike's involvement in the project has reduced. We synced offline and he agreed to go back to the reviewer role to help with first passes.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by alculquicondor** [comment]
cc @ahg-g

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -166,14 +166,14 @@ aliases:
     - alculquicondor
     - Huang-Wei
     - ahg-g
-    - damemi
```
Comment (damemi) [inline OWNERS_ALIASES:169]: why remove from maintainers? I'm fine being added to reviewers to get the auto-review requests but would also like to stay on as a maintainer if possible

Comment (alculquicondor) [reply OWNERS_ALIASES:169]: Synced offline: we have this soft policy where approvers are not reviewers and viceversa.

Added to emeritus for historical record.

Comment (damemi) [reply OWNERS_ALIASES:169]: sgtm, thanks!

**Review by damemi — COMMENTED** [review]
/lgtm

**Comment by alculquicondor** [comment]
ping @dims

**Comment by dims** [comment]
@alculquicondor Apologies, slipped through the cracks.

/approve

**Pull request merged**